README.md

```diff
@@ -11,13 +11,14 @@
 
 - [Supported Platforms](#supported-platforms)
 - [Installation](#installation)
-- [Methods](#methods)
+- [API](#api)
+- [Example](#example)
 
 <!-- /MarkdownTOC -->
 
 ## Supported Platforms
 
-- iOS
+- iOS (Xcode 12+)
```

Comment:
Why you need suffix `Xcode 12+`?

Reply:
Because I've added the `<framework src="AppTrackingTransparency.framework" weak="true" />` to `plugin.xml`, this will only work in Xcode 12+, earlier versions don't know about this framework.

Reply:
This section is about supported platforms not xcode editor version. Let’s remove the suffix to avoid confusion.

Reply:
Do you want to add this info somewhere?

Reply:
https://developer.apple.com/ios/submit/

> Starting April 2021, all iOS and iPadOS apps submitted to the App Store must be built with Xcode 12 and the iOS 14 SDK.

Ok, keep it as it is. Will be removed in 7 months